niworkflows/common/report.py

```diff
@@ -8,6 +8,13 @@
 import jinja2
 from pkg_resources import resource_filename as pkgrf
 
+from nipype.interfaces.base import File, traits
+
+class ReportCapableInputSpec(object):
+    generate_report = traits.Bool(
+        desc="Set to true to enable report generation for node"
+    )
+
```

Comment:
When I tried to multiclass with the ReportCapableInputSpec I got this error:
traits.trait_errors.TraitError: Cannot set the undefined 'generate_report' attribute of a 'FLIRTInputSpecRPT' object.

I couldn't figure out why the attribute wasn't being carried over into the flirt input spec so I reverted the changes.

Once this gets merged I'll open up a new PR focused on just that feature and see if the error can't be resolved there.